ci/rust_http_server/Cargo.toml

```diff
@@ -8,3 +8,5 @@ edition = "2021"
 [dependencies]
 hyper = { version = "=0.14.27", features = ["full"]}
 tokio = { version = "=1.31.0", features = ["full"]}
+
+[workspace]
```

Comment:
Is this necessary? I'd expect not, since it's a single crate with no dependencies on stuff in the parent workspace.

Reply:
Yes, I had to add it so that the expect scripts would run in CI